implants/lib/eldritch/src/crypto/decode_b64_impl.rs

```diff
@@ -0,0 +1,44 @@
+use anyhow::{anyhow, Result};
+use base64::{Engine, engine::general_purpose};
+
+pub fn decode_b64(content: String, encode_type: String) -> Result<String> {
```

Comment:
Can you set this to be optional and default to STANDARD?
Starlark will respect the Option<String> syntax